components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/PublishedNodesEntryModel.cs

```diff
@@ -26,6 +26,10 @@ public class PublishedNodesEntryModel {
         [DataMember(EmitDefaultValue = false, IsRequired = false)]
         public int? DataSetPublishingInterval { get; set; }
 
+        /// <summary> The Publishing interval for a dataset writer in timespan format </summary>
```

Comment:
Let's specify for `DataSetPublishingInterval` above whether it is in seconds or milliseconds.

Reply:
done